docs/reference/team-processes/triage-process.md

```diff
@@ -4,108 +4,130 @@ title: Triage Owner Duties
-* **Make sure the issue type is correct.** The default when filing in Jira or from GitHub is type “task” but it might need to be changed to “spike” if it’s a research task or work breakdown task, or “bug” in which case you will want to fill out the “Found in” (environment) and “[Severity][bug-severity]” (S1 is highest, S4 is lowest) fields.
-* **Add an initial estimated number of story points** if someone on the call can [estimate][point-estimation].
-* **If applicable, add the appropriate epic, labels, and/or issue links.** It’s uncommon for a new issue in triage to lack an epic if it should be associated with one, but it’s good to be aware of what epics exist and what the roadmap looks like. Some common labels to consider using are `maintenance` and  `qa-`, but be sure to check out [labels we use][labels-we-use] to be aware of others. Issue links create a link between issues and can be “blocked by X issue,” “relates to X issue,” and more.
-* **Move the issue out of the backlog section.** If the issue seems high priority or someone on the call asks to be assigned, it might belong in the current or following sprint (update the “sprint” field if so), or at the top of the “to do” list. Either way, drag and drop the task up and out of the backlog section (or manually change the status and refresh) and the issue status will change from “new” to “to do”.
+- **Make sure the issue is clear and ready to be worked on.** If more info or context is needed that an engineer in the triage meeting can provide, either leave a summary comment of what’s stated (if it’s quick) or ask them to comment on it. You can also @-mention those who may be able to answer pending questions. If more info is needed and the issue seems high priority, consider pinging the reporter directly about the issue after the triage meeting.
+- **Check that the issue type is correct.** The default type when filing in Jira is “task”. Other types include “spike” (for research task or work breakdown task) and “bug” (in which you will need to fill out the “Found in” (environment) and “[Severity][bug-severity]” (S1 is highest, S4 is lowest) fields).
+- **Add an initial estimated number of story points**, see [Estimation and Point Values][point-estimation].
+- **Add the appropriate epic, labels, and/or issue links, if applicable.**
+  - It’s uncommon for a new issue in triage to lack an epic if it should be associated with one, though it’s good to be aware of existing epics and the roadmap.
+  - Add the appropriate labels (e.g. `needsdiscussion`, `needslegal`, `needsproduct`, `needsux`). Other common labels to consider are `maintenance` and `qa-` (see [labels][labels-we-use] for more).
```

Comment:
Nit: no issue here other than I hardly see these labels applied. I'd maybe consider examples of the common ones we come across. Otherwise, clarify that this would be a team decision and not on the shoulders of the triage owner?

Reply:
I think if I clarified it for this point, I would also have to clarify it for every point on this list as well. Since this is the "Host the Triage Meeting" section (and from our TO experience), the team will help the triage owner fill any missing information in the ticket before it is moved out of the backlog.